tests/foreman/ui/test_contentview.py

```diff
@@ -2926,9 +2683,6 @@ def test_positive_delete_with_kickstart_repo_and_host_group(session, default_sat
                 'host_group.name': hg_name,
                 'host_group.lce': lc_env.name,
                 'host_group.content_view': content_view.name,
-                'host_group.content_source': sat_hostname,
-                'host_group.puppet_ca': sat_hostname,
-                'host_group.puppet_master': sat_hostname,
```

Comment:
please, can you explain why the default_sat was removed from the test here ? Only speedup ? do we test these host_group properties somewhere else?

Reply:
When `sat_hostname` was passed in, the UI shows empty in hostgroup creation so the test fails.  I'm not entirely sure what the setup is actually supposed to be in hostgroup for `sat_hostname` to appear in the selection.  @omkarkhatavkar do you have any idea?